test/suites/providers/aws.js

```diff
@@ -0,0 +1,18 @@
+// const { strictEqual } = require('assert');
+const AWSTransport = require('../../../src/providers/aws');
+
+describe('util fetch-data suite', () => {
+  it('should be able to create instance', () => {
+    const provider = new AWSTransport({
+      options: {
+        accessKeyId: 'AKIASS5V3WV23VA4KYOF',
+        accessKeySecret: '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',
```

Comment:
these keys must be revoked now as they were commited to open source code - they must be passed on via .env and never be commited

Reply:
removed